src/access/wac.ts

```diff
@@ -235,3 +264,52 @@ export function getGroupAccessAll(
 ): Promise<AgentWacAccess | null> {
   return getActorAccessAll(resource, getGroupAccessAllWac, options);
 }
+
+/**
+ * Set the Access modes for a given Agent to a given Resource.
+ *
+ * Important note: if the target resource did not have a Resource ACL, and its
+ * Access was regulated by its Fallback ACL, said Fallback ACL is copied to create
+ * a new Resource ACL. This has the side effect that the next time the Fallback
+ * ACL is updated, the changes WILL NOT IMPACT the target resource.
```

Comment:
Fixed in c6427b709730448fd8b03b519fb41b19400a5c11

Reply:
Haha, I saw the commit message - it would actually be hilarious if screen readers would actually SHOUT THIS at people. But I'm not sure if the actual behaviour if any better: it thinks it's an acronym, and will spell it out letter by letter :P